app/vendors/languages_lib.php

```diff
@@ -544,6 +544,7 @@ function __($string) {
                 'rue' => __d('languages', 'Rusyn', true),
                 'tah' => __d('languages', 'Tahitian', true),
                 'tly' => __d('languages', 'Talysh', true),
+                'mic' => __d('languages', 'Mi'kmaq', true),
```

Comment:
@RyckRichards, you need to use double quotes (`"Mi'kmaq"`), similarly to some other languages like: https://github.com/Tatoeba/tatoeba2/blob/dev/app/vendors/languages_lib.php#L525